ReSharper.FSharp/src/FSharp.Fantomas.Host/FSharp.Fantomas.Host.csproj

```diff
@@ -8,6 +8,14 @@
         <RootNamespace>JetBrains.ReSharper.Plugins.FSharp.Fantomas.Host</RootNamespace>
     </PropertyGroup>
 
+    <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+      <PlatformTarget>AnyCPU</PlatformTarget>
```

Comment:
Isn't it the default platform target? Assuming yes, is there any other place it gets overridden?

Reply:
I've just checked,  and `dotnet new console` creates `AnyCPU` for me:
<img width="770" alt="Screenshot 2021-05-11 at 15 52 11" src="https://user-images.githubusercontent.com/3923587/117818023-db48e180-b270-11eb-9bbf-36112c7205f0.png">

Reply:
Fantomas.Host.exe targets .NET 4.6.1, there is x86 by default